HIRS_ProvisionerTPM2/src/TPM2_Provisioner.cpp

```diff
@@ -70,6 +71,9 @@ int provision() {
                                        endorsementCredential,
                                        platformCredentials);
     identityClaim.set_client_version(CLIENT_VERSION);
+    string paccorOutputString =
+            RUN_PROCESS_OR_THROW("/opt/paccor/scripts/allcomponents.sh", "");
+    identityClaim.mutable_paccoroutput()->assign(paccorOutputString);
```

Comment:
Should be able to use `identityClaim.set_paccoroutput(paccorOutputString);`

Reply:
changed.